coursedata/adventures/de.yaml

```
@@ -603,164 +609,195 @@ adventures:
+                    if Form is Dreieck Winkel is 120
+                    else Winkel is 90
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
                     ```
                 start_code: |
-                    print 'Drawing Figures'
-                    figure is ask 'Do you want a square or a triangle?'
-                    if figure is triangle angle is 120 else angle is 90
-                    turn angle
+                    print 'Zeichne Formen'
+                    Form is ask 'Möchtest Du ein Quadrat oder ein Dreieck zeichnen?'
+                    if Form is Dreieck Winkel is 120 else Winkel is 90
+                    turn Winkel
                     forward 25
             8:
                 story_text: |
-                    ## Let's draw
-                    Now that we can repeat several lines, we can make figures more easily.
-                    We only have to set the angle once and then use that variable in the `repeat`.
+                    ## Lass uns zeichnen
+                    Jetzt, da wir einzelne Linien wiederholen können, fällt uns das Zeichnen einzelner Formen leichter.
+                    Wir setzen den Winkel einmalig und nutzen dann die Variable mit `repeat`.
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    angle = 90
+                    Winkel = 90
                     repeat 10 times
-                        turn angle
+                        turn Winkel
                         forward 50
                     ```
+                story_text_2: |
+                    ## Interaktiv
+                    Wir können auch das Programm verbessern, das verschiedene Formen zeichnet.
+                    Bekommst Du raus, wie weit die Schildkröte sich drehen muss?
+                    Vervollständige den Code und zeichne jedes Vieleck, das Du möchtest.
 
-                    ## Interaction
-                    Also, we can now improve the program that draws different figures.
-                    Can you figure out how far the turtle has to turn here? Finish the code and you can draw any polygon you'd like!
+                example_code_2: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    angles = ask 'How many angles should I draw?'
-                    angle = 360 / angles
-                    repeat angle times
+                    Ecken = ask 'Wie viele Ecken soll die Form haben?'
+                    Winkel = 360 / Ecken
+                    repeat Ecken times
```

As this isn't a list, maybe
```suggestion
                    Eckenanzahl = ask 'Wie viele Ecken soll die Form haben?'
                    Winkel = 360 / Eckenanzahl
                    repeat Eckenanzahl times
```
or
```suggestion
                    Anzahl_Ecken = ask 'Wie viele Ecken soll die Form haben?'
                    Winkel = 360 / Anzahl_Ecken
                    repeat Anzahl_Ecken times
```

---

> Ecken is the plural of Ecke.

Yes, and I appreciate that you used two different words ("Ecke" vs. "Winkel"), so children can distinguish when we mean "corner" (part of a shape) and when "angle" (a property/measure of a corner).

However, I think it can be confusing when we sometimes use plural variable names for variables that hold several things/values of said sort or type, and other times use plural variable names to hold a single number that indicates the number of such things. But as the English original doesn't make that distinction, we should maybe discuss it in general and/or for English first and until a decision is reached there stick to a more or less verbatim translation (i.e., keep it as-is without mentioning "Anzahl" in the variable names, even if the variable holds a number.)